### PR TITLE
Add ssm roles to README

### PR DESCRIPTION
**Pull request by lrrash**
This change adds a more detailed description to the README of all roles and permissions required to complete the multi-party approvals. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: ssm-automation/README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 This repository contains example SSM Automation Documents that can be used by SSM [Change Manager](https://docs.aws.amazon.com/systems-manager/latest/userguide/change-manager.html) feature to create PAAs and PAIs using multi-party authorization.
 
-**Note:** This process will require IAM Roles with permissions to access AWS Private CA. Before beginning, please make sure that you have an IAM role with the following trust relationship and permissions:
+## IAM Roles
+This process will require the creation of IAM Roles to approve the Change Manager Templates, Approve the Change Manager Requests, and execute the automation documents. Before beginning, please make sure that you have all of the following IAM roles.
```
Comment (hmphome) [inline ssm-automation/README.md:6]: probably second "Approve" doesn't need to start from a capital letter. Also, maybe we should combine the two into "approve the Change Manager Templates and Requests"?

File: ssm-automation/README.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+
+### CreatePAA and CreatePAI Roles
+These roles will be used to execute the automation documents which create the PAAs and PAIs.
```
Comment (hmphome) [inline ssm-automation/README.md:9]: which create`s` the ...

File: ssm-automation/README.md [hunk 3]
```diff
@@ -80,10 +146,11 @@ To create a PAA or PAI with multiple approvers follow these steps:
 5. Begin creating a new Template.
 6. For the Runbook option, choose the Automation Document you created.
 7. Choose the desired approvers and fill in the rest of the fields.
-8. Submit the Template for review. **Note:** Make sure that you have a "Template Reviewer" registered in the "Settings" tab of Change Manager.
-9. Approve the Template.
+8. Submit the Template for review. **Note:** Make sure that you have the Template Approver Role (described in the "Template Approver Role" section above) registered in the "Settings" tab of Change Manager under "Template Reviewer".
```
Comment (hmphome) [inline ssm-automation/README.md:149]: You can easily create links to headers, see https://www.markdownguide.org/extended-syntax/#heading-ids

File: ssm-automation/README.md
Comment (hmphome) [inline ssm-automation/README.md:1]: It looks like requiring a lot of manual work, could/should it be done by a CDK script instead?

**Review by hmphome — CHANGES_REQUESTED** [review]

File: ssm-automation/README.md (hunk 2 above)
Comment (lrrash) [reply ssm-automation/README.md:9]: I think its just 'create'

File: ssm-automation/README.md
Comment (lrrash) [reply ssm-automation/README.md:1]: Yes it probably should be eventually. Some SAs are currently creating a blog on how to do this setup though. The blog walks through setting up the roles and all the steps

File: ssm-automation/createRoles.sh [hunk 4]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Replace with your AWS account ID
+AWS_ACCOUNT="1234567890"
+
+# Replace with the number of approver roles you want to create
+NUMBER_OF_REQUEST_APPROVERS=2
+
+echo "Creating CreatePAARole..."
+aws iam create-role --role-name CreatePAARole --assume-role-policy-document file://policies/SSMTrustPolicy.json
```
Comment (hmphome) [inline ssm-automation/createRoles.sh:10]: Don't see policies/SSMTrustPolicy.json file, missing?

File: ssm-automation/createRoles.sh [hunk 5]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```
Comment (hmphome) [inline ssm-automation/createRoles.sh:1]: Let's make it safer, so it quits on any error in the middle. E.g.
```
set -euxo pipefail
```

**Review by hmphome — CHANGES_REQUESTED** [review]

File: ssm-automation/createRoles.sh (hunk 4 above)
Comment (lrrash) [reply ssm-automation/createRoles.sh:10]: Hmm somehow the policies didnt make it into the commit - I will add those

File: ssm-automation/README.md [hunk 6]
```diff
@@ -80,10 +158,11 @@ To create a PAA or PAI with multiple approvers follow these steps:
 5. Begin creating a new Template.
 6. For the Runbook option, choose the Automation Document you created.
 7. Choose the desired approvers and fill in the rest of the fields.
-8. Submit the Template for review. **Note:** Make sure that you have a "Template Reviewer" registered in the "Settings" tab of Change Manager.
-9. Approve the Template.
+8. Submit the Template for review. **Note:** Make sure that you have the [Template Approver Role](#template-approver-role) registered in the "Settings" tab of Change Manager under "Template Reviewer".
```
[the remark this answers is not in the document]
Comment (hmphome) [reply ssm-automation/README.md:163]: Okay, great GitHub auto-generate them! Thanks.

**Review by hmphome — APPROVED** [review]

**Pull request merged**